docs/scaling/OVERVIEW.md

```diff
@@ -0,0 +1,41 @@
+# Autoscaling
+
+There are three main components to the autoscaling system; autoscaler, activator and queue-proxy.
+
+## Queue-proxy
+Whenever a Serverless service is deployed by the user, the queue-proxy sidecar container is injected with it. This queue proxy does the following:
```

Comment:
```suggestion
Whenever a Serverless Service is deployed by the user, the queue-proxy sidecar container is injected with it. This queue proxy does the following:
```

(I can't actually remember what the conventions are on what should be capitalized, etc., but I think usually the CRDs get capitalized)